5/README.md

```diff
@@ -1,14 +1,17 @@
 ---
 domain: rfc.tango-controls.org
 shortname: 5/PROPERTY
-name:Property
+name: Property
 status: raw
-editor: Vincent Hardion (vincent.hardion@maxiv.lu.se)
+editor: Gwenaëlle Abeillé `<at synchrotron-soleil.fr - gwenaelle.abeille>`
+contributors:
+  - Vincent Hardion `<at maxiv.lu.se - vincent.hardion>`
+  - Reynald Bourtembourg `<at esrf.fr - bourtemb>`
 ---
 
-This document describes the Property,  a Tango concept representing an element of configuration in order to customise a Tango element. This document describes version 1.0 of the Property.
+This document describes the Property, a Tango concept representing an element of configuration in order to customise a Tango element. This document describes version 1.0 of the Property.
```

Comment:
Tango "element" is sort of undefined concept...

Reply:
Element could be any field of a Tango control system like Tango Device, Command, Attribute, System. or free value i.e. not a Tango object. What about this formulation:
```suggestion
This document describes the Property, a Tango concept representing one or more values to configure one of the following Tango elements namely Tango Device, Class, Attribute, System or a not associated to any Tango element, so called free properties. This document describes version 1.0 of the Property.
```

Reply:
Sounds good to me

Reply:
ok also for me.